rel/i18n/emqx_bridge_pulsar.hocon

```diff
@@ -43,6 +43,7 @@ authentication.label:
 buffer_memory_overload_protection.desc:
 """Applicable when buffer mode is set to <code>memory</code>
 EMQX will drop old buffered messages under high memory pressure.
+The high memory threshold is defined in config <code>sysmon.os.sysmem_high_watermark</code>.
```

Comment:
This one doesn't apply: `pulsar` doesn't call `load_ctl:is_high_mem()` like `wolff` does.

Reply:
then what applies?

Reply:
yeah, now that we're mentioning this, I noticed that `drop_if_highmem` is added to `replayq` options, but `replayq` doesn't do anything with it :melting_face: :see_no_evil:

Reply:
I suggest we fix it in another PR or we remove such config for pulsar